Dockerfile

```diff
@@ -1,7 +1,7 @@
-FROM debian:wheezy
+FROM gliderlabs/alpine:3.3
```

Comment:
I'd rather use `alpine:3.3` instead of `gliderlabs/alpine:3.3` just so it is an official image.